guides/plugins/apps/starter/starter-admin-extension.md

```diff
@@ -0,0 +1,155 @@
+    <meta>
+        <name>ListingExtension</name>
+        <label>Listing Extension App</label>
+        <description>This app extends the product listing</description>
+        <author>shopware AG</author>
+        <copyright>(c) shopware AG</copyright>
+        <version>1.0.0</version>
+        <license>MIT</license>
+    </meta>
+</manifest>
+
+```
+{% endcode %}
+
+## Set up communication between Shopware and the App
+
+Next, we need to set up an entry point, so Shopware and your app can communicate. The entry point is a static `.html` file, which includes the Extension SDK script and defines our extension.
+
+![Communication between the admin panel and your entry point](../../../../.gitbook/assets/extension-api-communication.png)
+
+The file will be rendered as a hidden iFrame within your admin panel. Using `postMessage` requests, the iFrame and your admin panel can communicate and exchange data.
+
+Let's create this file in a directory called `src`.
+
+```bash
+mkdir src
+touch src/index.html
+```
+
+{% code title="index.html" %}
+```html
+<!doctype html>
+<html>
+    <head>
+        <script src="https://unpkg.com/@shopware-ag/admin-extension-sdk/cdn"></script>
+    </head>
+    <script>
+        sw.notification.dispatch({
+            title: 'Hi there',
+            message: 'Looks like someone sent you a message'
+        });
+    </script>
+</html>
+
+```
+{% endcode %}
+
+This file contains example code that displays a simple notification within the administration. 
+
+### Start the local development server
+
+Next, we need to start the live server so you don't always have to reload the page manually.
+
+```bash
+npm install -g live-server
+live-server src
+```
+
+Now the file should be available on [http://127.0.0.1:8080](http://127.0.0.1:8080).
+
+### Add the entry point link to your manifest
+
+The final step of the setup is to configure your app to use that file as an entry point.
+
+In order to do that, we have to add an `admin` section to our `manifest.xml` file and pass it into the `base-app-url` tag:
+
+{% code title="manifest.xml" %}
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+        <!-- ... -->
+    </meta>
+    <admin>
+        <base-app-url>http://127.0.0.1:8080</base-app-url>
+    </admin>
+</manifest>
+```
+{% endcode %}
+
+Since the URL passed is only available locally, you will only be able to see changes on your own machine.
+If you want to share it, you need to host the entry point file somewhere or use services to expose local files as public URLs, such as [ngrok](https://ngrok.com/).
```

Comment:
Maybe qualify this as a development solution. For example, a CDN or other static site hosting can distribute the prod version.